include/irr/asset/ICPURenderpassIndependentPipeline.h

```diff
@@ -25,6 +25,8 @@ class ICPURenderpassIndependentPipeline : public IRenderpassIndependentPipeline<
     inline ICPUSpecializedShader* getShaderAtStage(E_SHADER_STAGE _stage) { return m_shaders[core::findLSB<uint32_t>(_stage)].get(); }
     inline ICPUSpecializedShader* getShaderAtIndex(E_SHADER_STAGE_IX _ix) { return m_shaders[_ix].get(); }
 
+	inline SPrimitiveAssemblyParams &getPrimitiveAssemblyParams() { return m_primAsmParams; }
```

Comment:
make also getters for other params structs

Reply:
make it a const, if you need mutable then make both const and non-const, but sometimes  mutability introduces non-trivial issues/constraints

Reply:
there are getters in IRenderpassIndependentPipeline with const qualifiers, a few
because I needed mutable, a did an overload on getPrimitiveAssemblyParams() in ICPURenderpassIndependentPipeline
Criss said that I can do it only in CPU files

Reply:
@devshgraphicsprogramming should I do it if there are actually getters in IRenderpassIndependentPipeline?

Reply:
>there are getters in IRenderpassIndependentPipeline with const qualifiers, a few

yea,I meant non-const getters ofc

Reply:
const getters in `IRenderpassIndependentPipeline` mutable getters in `ICPU....`

Reply:
resolved, see next commits